features/facebookImport/src/model/importers/posts-importer.js

```diff
@@ -0,0 +1,23 @@
+import MultipleFilesImporter from "./multiple-files-importer";
+import { MissingPostFilesException } from "./utils/failed-import-exception";
+import { removeEntryPrefix } from "./utils/importer-util";
+
+export default class PostsImporter extends MultipleFilesImporter {
+    _isTargetPostFile(entryName) {
+        const formattedEntryName = removeEntryPrefix(entryName);
+        return /posts\/your_posts_[1-9][0-9]?.json$/.test(formattedEntryName);
+    }
+
+    _importRawDataResults(facebookAccount, dataResults) {
+        dataResults.forEach((rawPosts) => {
+            const postsWithTimestamp = rawPosts.map((postData) => {
+                return { timestamp: postData.timestamp };
+            });
+            facebookAccount.posts.push(...postsWithTimestamp);
```

Comment:
We're assuming here a specific representation for `posts`, and maybe we shouldn't. Encapsulation dictates that we should have a public interface for this, maybe an "addPost" function, that would be independent of what's under it. If we decide to refactor this to have a map or anything else, like a set, we will have to refactor high and low.